lang/test/org/partiql/lang/eval/EvaluatorTestBase.kt

```diff
@@ -94,7 +111,7 @@ abstract class EvaluatorTestBase : TestBase() {
                 session,
                 compilerPipelineBuilderBlock
             )
-            AssertExprValue(resultForPermissiveMode, message = "(PERMISSIVE mode) Evaluated '$source' with evaluator ($message)")
+            AssertExprValue(resultForPermissiveMode, message = msg)
```

Comment:
It seems it can be safely removed. To ensure, I ran the tests while inflicting assertion failure and the number of tests failing with and without this block and their messages are identical. Thoughts?

Reply:
>To ensure, I ran the tests while inflicting assertion failure and the number of tests failing with and without this block and their messages are identical.
Based on this I also think it's safe to remove.